### ISSUE TITLE
YOLOv8 Commercial license

### ISSUE THREAD

**Issue by ksv87** [ultralytics/ultralytics#8146]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

YOLO provides pre-trained models on COCO and ImageNet datasets, can I use these models for commercial purposes, provided that I purchase a commercial license from YOLO?

**Comment by glenn-jocher** [comment]
@ksv87 hello! Yes, you can use our pre-trained models on COCO and ImageNet for commercial purposes with a commercial license. The license will grant you the flexibility needed for product development without the open-source requirements of the AGPL-3.0 license. If you need further assistance or have any more questions regarding the licensing process, feel free to reach out. Happy to help! 😊🚀

**Comment by ksv87** [comment]
@glenn-jocher thanks for the reply, will there be any problems with releasing ImageNet under license only for non-commercial research purposes?

**Comment by glenn-jocher** [comment]
@ksv87 yes, you can use the pre-trained YOLO models on COCO and ImageNet datasets for commercial purposes if you purchase a commercial license from Ultralytics. The commercial license provides more flexibility for commercial product development without the open-source requirements of the AGPL-3.0 license.

With a commercial license, you are typically allowed to embed Ultralytics software and AI models in commercial products and applications. It's important to review the specific terms and conditions of the commercial license to ensure compliance with its provisions.

For more information on obtaining a commercial license and understanding the terms, you should contact Ultralytics directly through their official channels. They will provide guidance on the licensing process and any associated costs.
